_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -0,0 +1,306 @@
+    auto_create_subnetworks          = false
+    gke_node_ip_range                = "10.1.0.0/20"
+}
+```
+
+Initialize the Terraform module.
+
+```bash
+terraform init
+```
+
+Apply the Terraform module.
+
+```bash
+tarraform apply
+```
+
+**Note:** The `terraform apply` may fail to complete.  In this case, please re-run `terraform apply`.
+
+### Deploy Forseti in an Existing GKE Cluster
+
+Create a file named *main.tf* in an empty directory and add the following content.  Add the appropiate values for each of the input vairables (e.g. domain, gsuite_admin_email).
+
+```bash
+module "forseti-on-gke-existing-gke-cluster" {
+    source                           = "terraform-google-modules/forseti/google//examples/on_gke_existing_gke_cluster"
+    credentials_path                 = ""
+    forseti_client_service_account   = ""
+    forseti_client_vm_ip             = ""
+    forseti_cloudsql_connection_name = ""
+    forseti_server_service_account   = ""
+    forseti_server_storage_bucket    = ""
+    gke_cluster_name                 = ""
+    gke_cluster_location             = ""
+    gke_service_account              = ""
+    project_id                       = ""
+    suffix                           = ""
+}
+```
+
+Initialize the Terraform module.
+
+```bash
+terraform init
+```
+
+Apply the Terraform module.
+
+```bash
+tarraform apply
+```
+
+## Deploying with Helm
+
+### Deploy Forseti in an Existing GKE Cluster (Helm)
+
+Forseti can be deployed on GKE without the use of Terraform if the following preconditions are met.
+
+1. Forseti has been deployed in a GCP project.
+2. A GKE cluster has already been deployed in GCP under the same organization as Forseti.
+3. The [GKE credentials](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) exist on the machine executing the deployement
+4. The [Helm binary](https://helm.sh/docs/using_helm/) is installed and initiallized on the machine executing the deployment.
+
+After these preconditions are met, add the *forseti-security-charts* Helm repo to your Helm environment.
+
+```bash
+helm repo add forseti-security https://forseti-security-charts.storage.googleapis.com/release
+```
+
+Create and download [IAM service account keys](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) in JSON format for the **forseti-server** and **forseti-client** service accounts.
+
+Follow the [chart installation instructions](https://github.com/forseti-security/helm-charts/tree/master/charts/forseti-security#installing-the-forseti-security-chart) to install Forseti on GKE.
+
+
+## Post Deployment Configuration Changes
+
+After deploying Forseti, the default server configuration defines
+
+```yaml
+    rules_path: /home/ubuntu/forseti-security/rules
+```
+
+This path doesn't exist in the container.  Instead, the value of `rules_path` must point to the rules path in the Forseti server GCS bucket.  The can be changed by executing the following.
```

Comment:
Should we also mention being able to change this in the Terraform forseti_server_conf.yaml and redeploying?

Reply:
Referencing this issue so we don't forget: https://github.com/forseti-security/forseti-security/pull/2992

Reply:
This point is moot if https://github.com/forseti-security/helm-charts/pull/16 gets merged.

Reply:
@hshin-g When you say, "being anle to change this in the Terraform forseti_server_conf.yaml", do you mean the one included in the Terraform module?

Reply:
For some reason I am only seeing that you changed the version number and logo on the PR you linked. Would you be able to link to the edits?

And yes- I do mean the one included in the Terraform module, as that is the yaml file that it reads, not this one.

Reply:
Somehow I didn't commit the file that was the reason for the PR.  Fixed.

Please consider the following scenarios.
1. The user may not be using Terraform to deploy Forseti on GKE.
2. The user may opt to not clone the module locally but reference it in Terraform Registry, in which case they will not have the config file local.